org-cyf-itp/hugo.toml

```
@@ -47,7 +47,7 @@ org = "https://github.com/CodeYourFuture"
 repo = "https://github.com/CodeYourFuture/curriculum/"
 pdrepo = "CYF-PD"
 root = "curriculum"
-orgapi = "https://api.github.com/repos/CodeYourFuture/"
+orgapi = "https://github-issue-proxy.illicitonion.com/repos/CodeYourFuture/" 
```

```suggestion
orgapi = "https://github-issue-proxy.illicitonion.com/cached/120/repos/CodeYourFuture/" 
```

There are a couple of stray settings in `issuesorgapi` in org-cyf-sdc/hugo.toml and org-cyf-guides/hugo.toml which I think are now unused?

---

Yeah I'll run over the rest now and update